Makefile

```
@@ -0,0 +1,9 @@
+UK_ROOT ?= $(PWD)/../../unikraft
+UK_LIBS ?= $(PWD)/../../libs
+LIBS := $(UK_LIBS)/lib-nettle:$(UK_LIBS)/lib-newlib
```

Libs do not usually have `lib-` in front of them if referring to the folder.
```suggestion
LIBS := $(UK_LIBS)/nettle:$(UK_LIBS)/newlib
```

---

That's how the repo's are named, with lib-. I cloned the repo's from the unikraft project.

---

I think it's an incosistency from `kraft`. Repo's are named with `lib-` and `app-`, but `kraft` removes that extension when cloning new libraries.

---

I'm not using kraft to build the app :)
But I'll do the change you requested on the master branch.

---

And I'll leave it like this for the staging one

---

No need to change it, just mention in the readme that the path must be adjusted.